dist/angular-svg-radial.js

```diff
@@ -0,0 +1,36 @@
+/*!
+ * angular-svg-radial 0.1.0
+ * (c) 2015 Vokal LLC
+ * https://github.com/vokal/angular-svg-radial
+ */
+angular.module("svgRadial", []).directive("radialProgress", function() {
+    "use strict";
+    return {
+        restrict: "E",
+        transclude: true,
+        replace: true,
+        template: '<div class="radial-progress">' + '<div class="base"></div>' + '<div class="radial">' + '<div class="piece"></div>' + '<div class="piece"></div>' + "</div>" + '<div class="transclude" ng-transclude></div>' + "</div>",
```

Comment:
What exactly is going on here, and why is it going on.

Is this like... a non-compressed-but-kinda-compressed dist file?